signtool/signtool.py

```diff
@@ -191,6 +191,9 @@ def sign(options, args):
         urls = options.format_urls[fmt][:]
         random.shuffle(urls)
 
+        if fmt in ("macapp", ):
+            fmt = "dmg"
+
```

Comment:
I saw coverage decrease with this patch, presumably because nothing tests the fmt changes. Do we *want* to test that?